pkg/workflows/management/create_install_eksa.go

```diff
@@ -47,7 +47,7 @@ func (s *installEksaComponentsOnWorkloadTask) Run(ctx context.Context, commandCo
 		commandContext.ClusterSpec.Cluster.ClearTinkerbellIPAnnotation()
 	}
 
-	logger.Info("Installing EKS-A custom components on workload cluster")
+	logger.Info("Installing EKS-A custom components on the management cluster")
```

Comment:
nit: initially the workload cluster in all these places meant wrt the kind cluster being the mgmt cluster vs the workload cluster getting created by the kind cluster.  

Looks fine to me if we want to make this change tho since by this time we already would've moved the capi components.

Reply:
I don't believe a new customer coming to eksa for the first time is going to grasp this implementation detail. They have asked for a management cluster and are now seeing logs about a workload cluster.

Reply:
Think of the cognitive load we're adding to customers just to understand this one log line. They have to understand quite a lot of implementation details of how we create clusters. I believe that's an unknown and unreasonable expectation.

Reply:
Yeah it looks fine to me since for the customer can understand this better. When we use this statement for debugging the engineer can probably figure out which cluster we are talking about. LGTM!